packages/config/config/devices/0x0258/nas-rp01z1.json

```diff
@@ -0,0 +1,144 @@
+{
+	"manufacturer": "Shenzhen Neo Electronics Co., Ltd",
+	"manufacturerId": "0x0258",
+	"label": "NAS-RP01Z1",
+	"description": "Neo Coolcam Repeater",
+	"devices": [
+		{
+			"productType": "0x0010",
+			"productId": "0x0716"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5
+		},
+		"2": {
+			"label": "Low Temperature Basic Set",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "High (?) Temperature Basic Set",
+			"maxNodes": 5
+		},
+		"4": {
+			"label": "Low Humidity Basic Set",
+			"maxNodes": 5
+		},
+		"5": {
+			"label": "High (?) Humidity Basic Set",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Red LED blink when measuring",
```

Comment:
I very recently added automatic checks if labels etc. comply with our style guide. This one is being highlighted.
I strongly suggest using [VSCode](https://code.visualstudio.com/) to edit these files as it will show you the errors and help you fix them, if it is set up correctly - see [Prerequisites](https://zwave-js.github.io/node-zwave-js/#/development/intro?id=prerequisites).
You may have to merge our `master` branch first. TL;DR:

1. Use VSCode
2. install recommended extensions when asked
3. merge our master branch: `git fetch upstream && git merge upstream/master`
4. make sure the environment is up to date: `yarn && yarn build`
5. restart VSCode for good measure

Then you'll get hints like these:
![grafik](https://github.com/zwave-js/node-zwave-js/assets/17641229/868030e4-1ee2-4772-b58a-1e053dc4c479)

Reply:
Let me outline what I have been doing.  I think it is working, with one quirk maybe you can help with.
1) Use github desktop in windows
2) Have a fork from master as personal repository. Periodically rebase the fork (currently 12.1.0) for this PR.
2A) Sometimes will rebase the PR after the fork has been rebased, if I get message about it missing (or old) something and therefore can't add a commit
3) After rebase of the fork, I run `yarn install` and `yarn build `from command line. I do not recall how yarn was installed in windows ;-(
4) Without VSC I would be lost, so no worries there. Reload packages as requested.
The only quirk I haven't figured out is when I hover over an `$import` I do not always see the imported details.  Sometimes just `"Used to import another config file or parts thereof"`.  So I have to flip between the files.

Reply:
> The only quirk I haven't figured out is when I hover over an $import I do not always see the imported details. 

I think you need to hover over the part after the `#`. Or install the https://github.com/zwave-js/vscode-config-editor/ extension if you haven't already, and use the preview (cursor is in one of the parameters):
![grafik](https://github.com/zwave-js/node-zwave-js/assets/17641229/450251c7-7d99-41ac-9233-e3b5e8c80603)

Reply:
I do have the vscode-config-editor installed. 

After more playing around, if I open VSC from Github desktop, no hover info.  If I open VSC directly, hover works fine.  Need to figure that out..., but can work with it for now. 

Also FYI I only use Windows for the PR work. I thought I would try to help on some of the `Help Wanted` or `Config `issues, to learn more about everything works, hope that is okay. There is still good weather here, so need to split my time on home outdoor projects.

My Zwave networks are on docker ZUI:latest on Pi's.